libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -163,8 +172,18 @@ uint8_t eeprom_read_byte(const uint32_t pos)
   */
 void eeprom_write_byte(uint32_t pos, uint8_t value)
 {
+#if defined(DATA_EEPROM_BASE)
```

Comment:
Here there will be a misalignment between the buffered eeprom value an the real value.

Reply:
when reading the EEPROM, align eeprom_buffer[pos] = (uint8_t)data;